### ISSUE TITLE
example teleport and stochastic_swap are broken. 

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/qiskit-terra#2549]
### Information

- **Qiskit Terra version**: master
- **Python version**:
- **Operating system**:

### What is the current behavior?

Examples give error `AttributeError: 'tuple' object has no attribute 'register'`

### What is the expected behavior?

Examples to run

**Comment by jaygambetta** [comment]
if we do #970 we will notice this earlier

**Comment by jaygambetta** [comment]
personally I think we should remove stochastic swap as an example as it is really a test.

**Comment by mtreinish** [comment]
It is really a test, I wrote it as a way to test the built binary wheels as part of the release process (since stochastic swap is the compiled code) it is necessary to be around for the build automation. I could not use the unit tests directly because they makes assumptions as running from the root of the repo which conflicts with how the build automation works. If you want to move it somewhere else that's fine but you'll have to update the CIBW_TEST_CMD variables in the travis and appveyor configs.

**Comment by jaygambetta** [comment]
No if there is a reason but it fails for me at the moment

**Comment by mtreinish** [comment]
Right, it only runs when we push a tag for a release, which we haven't done on master since #2414 broke things. We still need #970 to catch the breakages before they happen. I was just saying don't delete the file because it's used by the release automation to verify the binaries work.

**Comment by jaygambetta** [comment]
oh ok then im a bit confused -- maybe this should not be in examples then if it is suppose to not work unless a tag is release. 

To be honest I am a bit confused -- maybe some documentation for the examples folder is needed

**Comment by mtreinish** [comment]
The examples folder is just where I ended up putting it, mostly because we were using an example before it was added in: https://github.com/Qiskit/qiskit-terra/pull/2251. It can live anywhere honestly, the only requirements are that it's not in the sdist (so nowhere under `qiskit/`) and it can't be in `test/python` since it doesn't need to be imported by discovery. I'll gladly move it outside of examples if that's what you think makes sense.

That being said I'll push up a patch to fix #970 now (and address this in that too) but if we do end up moving the stochastic swap example script somewhere we should make sure it's included in that test run. Otherwise we'll be in for a surprise at release time...
